activemq-client/src/main/java/org/apache/activemq/util/ClassLoadingAwareObjectInputStream.java

```diff
@@ -40,7 +40,7 @@
     private final ClassLoader inLoader;
 
     static {
-        serializablePackages = System.getProperty("org.apache.activemq.SERIALIZABLE_PACKAGES","org.apache.activemq,org.fusesource.hawtbuf,com.thoughtworks.xstream.mapper").split(",");
+        serializablePackages = System.getProperty("java.lang,org.apache.activemq.SERIALIZABLE_PACKAGES","org.apache.activemq,org.fusesource.hawtbuf,com.thoughtworks.xstream.mapper").split(",");
```

Comment:
There's a typo here, it should be: "java.lang","

Reply:
Oh yes, my bad ;) I'm fixing.